docs/reporting.rst

```diff
@@ -0,0 +1,164 @@
+analyses. This page describes the plots forming the reports and well as
+infromation on how to take advantage of the interactive functionalities.
+You can also play around with `our demo`_.
+
+.. _our demo: https://me-ica.github.io/tedana-ohbm-2020/
+
+Report Structure
+================
+
+The image below shows a representative report, which has two sections: a) the summary view,
+and b) the individual component view.
+
+.. image:: /_static/rep01_overallview.png
+  :align: center
+Summary View
+------------
+This view provides an overview of the decomposition and component
+selection results. It includes four different plots.
+
+* **Kappa/Rho Scatter:** This is a scatter plot of `Kappa` vs. `Rho` features for all components.
+  In the plot, each dot represents a different component. The x-axis represents the kappa feature, and the
+  y-axis represents the rho feature. These are two of the most
+  informative features describing the likelihood of the component
+  being BOLD or non-BOLD. Additional information is provided via color
+  and size. In particuale, color informs about its classification
+  status (e.g., accepted, rejected); while size relates to
+  the amount of variance explained by the component (larger dot,
+  larger variance).
+.. image:: /_static/rep01_kapparhoScatter.png
+  :align: center
+  :height: 400px
+
+* **Kappa Scree Plot:** This scree plots provides a view of the components ranked by `Kappa`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_kappaScree.png
+  :align: center
+  :height: 400px
+
+* **Rho Scree Plot:** This scree plot provides a view of the components ranked by `Rho`.
+  As in the previous plot, each dot represents a component. The color of the dot informs us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_rhoScree.png
+  :align: center
+  :height: 400px
+
+* **Variance Explained Plot:** This pie plot provides a summary of how much variance is explained
+  by each individual component, as well as the total variance explained by each of the three
+  classification categories (i.e., accepted, rejected, ignored). In this plot, each component is
+  represented as a wedge, whose size is directly related to the amount of variance explained. The
+  color of the wedge inform us about the classification status of the component. For this view,
+  components are sorted by classification first, and inside each classification group by variance
+  explained.
+.. image:: /_static/rep01_varexpPie.png
+  :align: center
+  :height: 400px
+
+Individual Component View 
+-------------------------
+This view provides detailed information about an individual 
+component (selected in the summary view, see below). It includes two different
+interactive plots.
```

Comment:
```suggestion
component (selected in the summary view, see below).
```

Reply:
Instead here we should mention the PNG views and showcase one (or two, if you want to show idealized "accepted" and "rejected")